configs/components/fesom/fesom.yaml

```diff
@@ -1,57 +1,79 @@
 - '1.4-recom-mocsy-slp'
-- 2.0-paleodyn
-- 1.4-recom-awicm
 - '2.0'
+- '2.0-o'
+- '2.0-paleodyn'
 - '2.0-r' # OG: temporarily here
+- '2.1'
 - '2.1-recom'
 - '2.1-wiso'
-- '2.1'
 
 choose_version:
-  '1.4-recom-awicm':
-    destination: fesom-1.4
-    branch: co2_coupling
-  '1.4-recom-mocsy-slp':
-    branch: fesom-recom-mocsy-slp
-    destination: fesom-1.4
-  '1.4-esm_interface':
-    destination: fesom-1.4
-    branch: using_esm-interface
+    '1.4-esm_interface':
+        branch: using_esm-interface
+    '1.4-recom':
+        branch: master
+        # add recom variables to fesom1 output scheduler; see components/recom/recom.yaml
+        add_yearly_outputs:
+            - "${recom.yearly_outputs}"
+        add_monthly_outputs:
+            - "${recom.monthly_outputs}"
+        add_daily_outputs:
+            - "${recom.daily_outputs}"
+        add_12_step_outputs:
+            - "${recom.12_step_outputs}"
+    '1.4-recom-awicm':
+        branch: fesom_recom_1.4_master
+        # add recom variables to fesom1 output scheduler; see components/recom/recom.yaml
+        add_yearly_outputs:
+            - "${recom.yearly_outputs}"
+        add_monthly_outputs:
+            - "${recom.monthly_outputs}"
+        add_daily_outputs:
+            - "${recom.daily_outputs}"
+        add_12_step_outputs:
+            - "${recom.12_step_outputs}"
+    '1.4-recom-mocsy-slp':
+        branch: fesom-recom-mocsy-slp
 
 git-repository: https://gitlab.dkrz.de/modular_esm/fesom-1.4.git
 install_bins: bin/fesom
 clean_command: ${defaults.clean_command}
 comp_command: mkdir -p build; cd build; cmake ..;   make install
 
 metadata:
-        Institute: Alfred Wegener Institute
+        Institute: Alfred Wegener Institute for Polar and Marine Research (AWI)
         Description:
                 Multiresolution sea ice-ocean model that solves the equations
                 of motion on unestructured meshes
         Authors: Dmitry Sidorenko (Dmitry.Sidorenko@awi.de), Nikolay V. Koldunov (nikolay.koldunov@awi.de)
         Publications:
-                - "The Finite-volumE Sea ice-Ocean Model (FESOM2) <https://doi.org/10.5194/gmd-10-765-2017>"
-                - "Scalability and some optimization of the Finite-volumE Sea ice-Ocean Model, Version 2.0 (FESOM2) <https://doi.org/10.5194/gmd-12-3991-2019>"
+                - "Danilov et al. 2004: A finite-element ocean model: principles and evaluation <https://doi.org/10.1016/s1463-5003(02)00063-x>"
+                - "Wang et al. 2014: The Finite Element Sea Ice-Ocean Model (FESOM) v.1.4: formulation of an ocean general circulation model. <https://doi.org/10.5194/gmd-7-663-2014>"
         License:
-                Please make sure you have a licence to use FESOM. In case you are unsure,
-                please contact redmine...
+                www.fesom.de
 
 restart_rate: "12"
 restart_unit: "m"
 restart_first: 12
 restart_flag: "last"
+yearly_outputs_start: 1 # Year in which the yearly output starts to be saved
```

Comment:
the default in the `AWI_FESOM_YAML` was `1000` in `release`. Why is it okay/better to keep it 1? Will this affect other people running fesom-1.4 simulations?

Reply:
I assume that was decided to save disk space. I dont know who decided that. I think the default output block for yearly outputs should also start in the first year since this is what you would assume, right?
If a user wants special settings (like 'save yearly outputs only after the 1000th model year'), the user can simply edit this block.

Reply:
Okay, thanks for the info :)